index.html

```diff
@@ -5,22 +5,182 @@
+        <div class="header-top">
+            <div class="wrapper">
+                <nav>
+                    <ul class="top-nav">
+                        <li><img src="images/icons/fb-icon.svg" alt="fb-icon"></li>
+                        <li><img src="images/icons/twitter-icon.svg" alt="twitter-icon"></li>
+                        <li>English</li>
+                        <li>My Page</li>
+                        <li>Logout</li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+
+        <div class="header-main">
+            <div class="wrapper">
+                <nav class="main-nav">
+                    <div class="icon-nav">
+                        <a href="index.html"><img src="images/logo.jpg" alt="logo image"></a>
+                    </div>
+                    <div class="icon-menu-mobile">
+                        <img src="images/icons/hamburger-icon.svg" alt="Hamburger icon">
+                    </div>
+                    <ul class="bottom-nav">
+                        <li><a  class="active" href="about.html">About</a>
+                        <li><a href="#program">Mission</a></li>
+                        <li><a href="#testimonials">Testimonials</a></li>
+                        <li><a href="#partners">Partners</a></li>
+                        <li><a href="#">Blog</a></li>
+                        <li><button type="button">Follow US</button></li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+
+        <div class="nav-mobile">
+            <div class="cross">
+                <img src="images/icons/cross.svg" alt="cross">
+            </div>
+            <ul class="container">
+                <li><a href="index.html">Home</a>
+                <li><a href="about.html">About</a>
+                <li><a href="#program">Mission</a></li>
+                <li><a href="#testimonials">Testimonials</a></li>
+                <li><a href="#partners">Partners</a></li>
+                <li><a href="#">Blog</a></li>
+            </ul>    
+        </div>
+    </header>
+
+    <main>
+        <section id="headline">
+            <div class="wrapper headline-content">
+                <span>"Coding Con"</span>
+                <h1>live Coding <br/>Pair-to-Pair programming <br/>is the new  TREND</h1>
+                <div class="headline-description">
+                    <p>Pair programming is an software development technique in which two programmers work together at one workstation. One, the driver, writes code while the other, the observer or navigator, reviews each line of code as it is typed in. The two programmers switch roles frequently.</p>
+                </div>
+                <div class="headline-footer">
+                    <h2>Upcoming Event - Unlimited Seats (28 ~ 31 March 2023)</h2>
+                    <h3>@USA - Cal Tech University</h3>
+                    <h3> <<<i> with @ahroniy </i>>></h3>
+                </div>
+            </div>
```

Comment:
- [ ] As requested by the previous reviewer, From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as much as possible but as you can see if comparing the two designs below they do not match.
   - As you can see from Cindy's design the background image does not cover the whole of this section.
   - The design of the `Pair-to-Pair programming <br/>is the new  TREND` in your project should look like the `CREATIVE COMMON GLOBAL SUBMIT 2015`
   - This section should have some background design as Cindy designed.
### Your design
![Screenshot from 2023-01-07 02-13-58](https://user-images.githubusercontent.com/92303891/211127000-6846b4ed-22bc-4747-91fd-7ea500f84c76.png)

### Prefered design
![image](https://user-images.githubusercontent.com/92303891/211126750-1c37ca0e-b9f3-4b19-ae84-18d4ba388384.png)

Reply:
✅ Did the Heading as design